tests/demo/test_demo_mediators.py

```diff
@@ -104,18 +72,25 @@ async def test_demo_mediators_unknown_to_sender(
 
     # BOB MEDIATOR 1: re-wrap to a new mediator
     forward_bob_1 = await unpack_forward(
-        resolvers_config_mediator_1, pack_result.packed_msg
+        resolvers_config_mediator1, pack_result.packed_msg
     )
+
     forward_bob_2 = await wrap_in_forward(
+        resolvers_config=resolvers_config_mediator1,
         packed_msg=forward_bob_1.forwarded_msg,
-        routing_key_ids=["mediator2-routing-key-id"],
-        forward_headers=[{"expires_time": 99999}],
-        resolvers_config=resolvers_config_mediator_1,
+        # FIXME dict-to-obj (from_dict) for forward body doesn't work now
+        to=forward_bob_1.forward_msg.body["next"],
+        enc_alg_anon=PackEncryptedConfig().enc_alg_anon,
```

Comment:
Can we have the same default value for `enc_alg_anon` as in `PackEncryptedConfig`?

Reply:
done